server/game/cards/06-WoE/PrimordialVault.js

```diff
@@ -0,0 +1,43 @@
+const Card = require('../../Card.js');
+
+class PrimordialVault extends Card {
+    // At the start of your turn, make a Cultist.
+    //
+    // Action: Destroy 4 friendly Cultists. If you do, search your
+    // deck and discard pile for Tangaika, reveal it, add it to your
+    // hand, and shuffle your deck.
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onBeginRound: (_, context) =>
+                    context.player === this.game.activePlayer &&
+                    context.player.tokenCard.name === 'Cultist'
```

Comment:
should check if context.player.tokenCard is defined
(context.player.tokenCard && context.player.tokenCard.name === ) just in case someone takes control of this artifact in a non-WoE deck

Reply:
Good call, done (and fixed in Blorb Hive too).